src/index.js

```diff
@@ -10,6 +10,8 @@ const PsModuleMap = require('./PsModuleMap');
 const Psc = require('./Psc');
 const PscIde = require('./PscIde');
 const dargs = require('./dargs');
+const execSync = require('child_process').execSync
```

Comment:
I did notice you're using `cross-spawn`, and did honest try to run `psc-package` asynchronously, but ran into weird synchronization issue and decided to fall back to `execSync`. What I didn't try is to use `cross-spawn`'s `sync`. On it.